### ISSUE TITLE
Add submission guidelines from Docs-Diversity-Inclusion-Badging

### ISSUE THREAD

**Issue by bistaastha** [badging/diversity-and-inclusion#4]
Currently, submission guidelines for Events and Projects are listed at:
https://github.com/Nebrethar/Docs-Diversity-Inclusion-Badging

Those can either be added to the README of this repository or as separate files here. Another approach would be to put these guidelines on respective [Event]{https://github.com/badging/event-diversity-and-inclusion/} and [Project]{https://github.com/badging/project-diversity-and-inclusion}.

@badging/maintainers Where would these fit better?

**Comment by Nebrethar** [comment]
I believe that any information regarding exclusively Event or Project requirements can be most easily found in their own respective repositories.

TL;DR I recommend Event and Project requirements in Event and Project repos
